File: code/server/src/Core/Entities/Mueble.cs
using IMT_Reservas.Server.Core.Abstraction;

namespace IMT_Reservas.Server.Core.Entities;

public class Mueble : Entity
{
    public string Nombre { get; set; } = string.Empty;
    public string? Tipo { get; set; }
    public string? Ubicacion { get; set; }
    public int? IdAmbiente { get; set; }
    public Ambiente? Ambiente { get; set; }
    public int NumeroGaveteros { get; set; }
    public double? Longitud { get; set; }
    public double? Profundidad { get; set; }
    public double? Altura { get; set; }
    public double? Costo { get; set; }
}

File: code/client/src/app/entities/admin/model/mueble.ts
import { BaseModel } from '@shared/model';
export class Muebles extends BaseModel {
  Nombre: string | null = null;
  NumeroGaveteros: number | null = null;
  Ubicacion: string | null = null;
  IdAmbiente: number | null = null;
  NombreAmbiente: string | null = null;
  Tipo: string | null = null;
  Costo: number | null = null;
  Longitud: number | null = null;
  Profundidad: number | null = null;
  Altura: number | null = null;
  constructor() {
    super();
    this.Nombre = null;
    this.NumeroGaveteros = null;
    this.Ubicacion = null;
    this.IdAmbiente = null;
    this.NombreAmbiente = null;
    this.Tipo = null;
    this.Costo = null;
    this.Longitud = null;
    this.Profundidad = null;
    this.Altura = null;
  }
}
